### ISSUE TITLE
Packaging error for Android ASTC / Client

### ISSUE THREAD

**Issue by alexhajdu** [FrozenStormInteractive/Unreal-SignalR#8]
Hey devs, I've got a packaging error for Android ASTC (arm64).

```
UATHelper: Packaging (Android (ASTC)):     15 errors generated.
PackagingResults: Error: invalid application of 'sizeof' to an incomplete type 'FSignalRValue'
PackagingResults: Error: no type named 'ConstPointerType' in 'TCallTraits<FSignalRValue>'
PackagingResults: Error: no type named 'ConstInitType' in 'TTypeTraits<FSignalRValue>'
PackagingResults: Error: no type named 'KeyInitType' in 'TDefaultMapHashableKeyFuncs<FString, FSignalRValue, false>'
PackagingResults: Error: no type named 'ElementInitType' in 'TDefaultMapHashableKeyFuncs<FString, FSignalRValue, false>'
PackagingResults: Error: no type named 'KeyType' in 'TDefaultMapHashableKeyFuncs<FString, FSignalRValue, false>'
PackagingResults: Error: no type named 'KeyType' in 'TDefaultMapHashableKeyFuncs<FString, FSignalRValue, false>'
PackagingResults: Error: no member named 'bAllowDuplicateKeys' in 'TDefaultMapHashableKeyFuncs<FString, FSignalRValue, false>'
PackagingResults: Error: no type named 'KeyInitType' in 'TSortableMapBase<FString, FSignalRValue, FDefaultSetAllocator, TDefaultMapHashableKeyFuncs<FString, FSignalRValue, false> >'
PackagingResults: Error: no type named 'KeyConstPointerType' in 'TSortableMapBase<FString, FSignalRValue, FDefaultSetAllocator, TDefaultMapHashableKeyFuncs<FString, FSignalRValue, false> >'
PackagingResults: Error: invalid range expression of type 'const TMap<FString, FSignalRValue, FDefaultSetAllocator, TDefaultMapHashableKeyFuncs<FString, FSignalRValue, false> >'; no viable 'begin' function available
PackagingResults: Error: no member named 'Add' in 'TMap<FString, FSignalRValue, FDefaultSetAllocator, TDefaultMapHashableKeyFuncs<FString, FSignalRValue, false> >'
PackagingResults: Error: implicit conversion of nullptr constant to 'bool' [-Werror,-Wnull-conversion]
PackagingResults: Error: implicit conversion of nullptr constant to 'bool' [-Werror,-Wnull-conversion]
```

**Comment by intelligide** [comment]
It should be fixed with the commit d29e01f

**Comment by alexhajdu** [comment]
Yes, removing the anonymous union with `TVariant` did the trick.
